dev-docs/modules/dynamicAdBoostRtdProvider.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: page_v2
+title: Dynamic Ad Boost Real Time Data Provider
+display_name: Dynamic Ad Boost Real Time Data Provider
+description: Dynamic Ad Boost Real Time Data Provider
+page_type: module
+module_type: rtd
+module_code: dynamicAdBoostRtd
+enable_download: false
```

Comment:
@milosandrejic setting this to `true` will reveal the module in the list.

The module it self is available: https://docs.prebid.org/dev-docs/modules/dynamicAdBoostRtdProvider